LCD_digits/digits/LCDNumber.js

```diff
@@ -0,0 +1,51 @@
+const LCDDigit = require("./LCDDigit").LCDDigit;
+
+class LCDNumber {
+    constructor(number) {
+        this.number = number;
```

Comment:
the property `this.number` is not being used anywhere so you can just use the number that's passed in.